CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.33.0
 
-* n/a
+* Moved from using the Jaeger exporter to OTLP exporter by default
```

Comment:
Maybe this deserves a section about deprecations and breaking changes to note the Jaeger OTel client removal? Especially since the user actually needs to change the URL etc where the traces are send or upgrade Jaeger server etc.

Reply:
Yes, good point. What about it now?

Reply:
Should it also cover that the OTel Jaeger client is not part of the images anymore and if soeone wants to use it they need to add the binaries?

Reply:
good point. just added.